card/mod/05_standard/set/self/admin_info.rb

```diff
@@ -0,0 +1,26 @@
+view :core do |args|
```

Comment:
Unused block argument - `args`. You can omit the argument if you don't care about it.